src/main/java/bio/overture/ego/controller/TokenController.java

```diff
@@ -98,9 +98,12 @@ public TokenController(@NonNull TokenService tokenService) {
     return format("Token '%s' is successfully revoked!", token);
   }
 
-  @ResponseBody
-  List<TokenResponse> listTokens(@RequestHeader(value = "Authorization") String authorization) {
-    return null;
+  @RequestMapping(method = RequestMethod.GET, value = "/list_token")
```

Comment:
lets rename this to just token.

This way we get proper REST usaged with GET, POST, and DELTE for /token.

I realize now the check_token endpoint must be confusing you. The reason that exists is for backwards compatibility with ICGC.